index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require_once 'functions.php';
+
+$db = getDb();
+$albums = retrieveAlbums($db);
+shuffle($albums);
+$displayalbums = displayAlbums($albums);
+
+?>
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Some Good Albums</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Inknut+Antiqua:wght@600&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+
+    <header>
```

Comment:
remove any story 2 related functionality until you have built the whole thing. aiming for deployable increments between stories

Reply:
removed form file and buttons